sdk/communication/Shared/tests/CommunicationTestEnvironment.cs

```diff
@@ -26,7 +26,7 @@ public class CommunicationTestEnvironment : TestEnvironment
 
         public string LiveTestStaticConnectionString => GetRecordedVariable(
                 LiveTestStaticConnectionStringEnvironmentVariableName,
-                options => options.HasSecretConnectionStringParameter("accessKey", SanitizedValue.Base64));
+                options => options.IsSecret("endpoint=https://sanitized.communication.azure.com/;accesskey=Kg=="));
```

Comment:
I am a bit confused about this one, I believe, we use DynamicConnectionString in our tests.
Also, this is from Shared folder, so applicable for all Communication SDKs, are we sure this is not breaking anything? Or is it some tech debt being removed?